include/grpcpp/server_builder_impl.h

```diff
@@ -257,6 +271,16 @@ class ServerBuilder {
   /// at any time.
   experimental_type experimental() { return experimental_type(this); }
 
+  enum ExternalConnectionType {
+    CONNECTION_FROM_FD = 0  // in the form of a file descriptor
+  };
+  // EXPERIMENTAL API:
+  // Create an acceptor to take in external connections and pass them to the
+  // gRPC server.
+  std::unique_ptr<grpc::ExternalConnectionAcceptor>
+  AddExternalConnectionAcceptor(ExternalConnectionType type,
```

Comment:
shouldn't this be a part of experimental_type?

Reply:
Done.